packages/twenty-server/src/engine/core-modules/jwt/services/jwt-wrapper.service.ts

```diff
@@ -167,4 +168,24 @@ export class JwtWrapperService {
       return ExtractJwt.fromUrlQueryParameter('token')(request);
     };
   }
+
+  isTokenExpired(token: string, expirationBufferInSeconds = 5 * 60): boolean {
```

Comment:
let's extract this to an util, its nice to have this tooling but we shouldn't have to instantiate a service and to import the module when we want to use it (also easier to add unit test).

isTokenExpiredOrInvalid would be a more proper naming given the behavior with exceptions